engine/src/main/java/io/camunda/zeebe/engine/api/records/ImmutableRecordBatchEntry.java

```diff
@@ -42,4 +42,8 @@ public interface ImmutableRecordBatchEntry {
    *     its maximum size
    */
   int getLength();
+
+  default boolean shouldSkipProcessing() {
+    return false;
```

Comment:
I added this because 8.1 doesn't have `LogAppendEntry` which we used in 8.2 to mark entries that should be skipped